pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.black]
+line-length = 80
+target-version = ['py39']
```

Comment:
Based on [this](https://github.com/psf/black/issues/751#issuecomment-473066811), I think this should actually be our full supported version range, which is currently 3.7, 3.8, 3.9, 3.10, and 3.11.

Reply:
Good eyes. Will update this.

Reply:
Just a note: after updating the branch and regenerating the code, I redid the whole process of code regeneration again with updated target-version. It had no effect: https://github.com/grpc/grpc/compare/1b68c728ece6b6f3da5e9fcee22de051df396282..75a1b832e4660ae7f51633bc79d552840b02e55b